source/educators/concepts/communication/notifications_preferences.rst

```diff
@@ -10,185 +10,180 @@ Notifications & Preferences
+- Turn tray notifications on or off.
+- Turn email notifications on or off.
+- Choose an email cadence: Immediate, Daily, or Weekly
 
-.. list-table::
-   :widths: 25 25 25 25
-   :align: center
-   :header-rows: 1
-
-   * - Preference
-     - Default Tray Preference
-     - Default Email Preference
-     - Visibility
-   * - Activity notifications
-     - ON
-     - ON / Daily
-     - All
-   * - New discussion posts
-     - OFF
-     - OFF
-     - All
-   * - New question qosts
-     - OFF
-     - OFF
-     - All
-   * - New posts from instructors
-     - ON
-     - ON
-     - All
-   * - Reported content
-     - ON
-     - ON / Daily
-     - Forum moderators
-   * - Course updates
-     - ON
-     - OFF
-     - All
-   * - Essay assignment grade received
-     - ON
-     - ON / Daily
-     - All
-   * - New ORA submission for staff grading
-     - ON
-     - OFF
-     - Course staff, Course admin
 
-.. _Notifications Grouping:
+.. _notifications-grouping-educators:
 
 Notifications Grouping
-######################
+======================
 
-High-volume notifications, such as new discussion posts or new ORA submissions, can clutter the notification tray. This clutter reduces the visibility of other important notifications, making it challenging for users to stay informed effectively. As a result, crucial but potentially high-volume notifications need to be turned off by default to avoid overwhelming users. The notification grouping feature mitigates this issue by combining similar notifications into a single, unified notification, based on their context. Currently, grouping is supported for two notification types:
+Potentially high volume notifications, such as new discussion posts and new ORA submissions,
+can clutter the notification tray. The notification grouping feature mitigates this issue by combining
+similar notifications into a single one, based on their context. Currently, grouping is supported for
+two notification types:
 
 - New discussion posts: Notifications are grouped by the specific course.
-- New ORA submissions: Notifications are grouped by the ORA assignment ID.
+- New ORA submission for staff review: Notifications are grouped by the ORA ID.
 
-.. image:: /_images/educator_concepts/Notificaitons_grouping_ORA_screenshot.png
+.. image:: /_images/educator_concepts/Notifications_grouping_ORA.png
   :width: 400
   :align: center
   :alt: Screenshot of grouped and ungrouped ORA submission received notification.
 
-.. _Notifications Expiry:
+.. _notifications-expiry-educators:
 
 Notifications Expiry
-######################
+====================
 
-Everyday, notifications older than 60 days are deleted from the database to keep the storage in control. Developers can change this duration from the django settings page.
+Your site administrator may set a time limit for how long notifications remain available.
+Once they reach that age, they are automatically removed from your notification tray.
```

Comment:
Please add a `.. seemore::` and a `**Maintenance Table**` - see other docs pages for examples

Reply:
I would add links to any other docs relevant to notifications (educator or learner's docs), any pages that were referenced within the text, and possibly even a link to a page on Discussions